adapter-excel/src/main/java/ch/prevo/open/node/adapter/excel/AbstractNotificationWriter.java

```diff
@@ -162,12 +163,21 @@ private void createHeading(Row row, String label, CellStyle headingStyle) {
 
     @Override
     public void close() throws IOException {
+        adjustColumnWidths(COMMENCEMENTS_LABEL);
+        adjustColumnWidths(TERMINATION_LABEL);
         try (OutputStream fileOut = new FileOutputStream(filename)) {
             workbook.write(fileOut);
         }
         workbook.close();
     }
 
+    private void adjustColumnWidths(String sheetName) {
```

Comment:
Suggestion for rename: `adjustColumnWidthsInSheet(String sheetName)`

Reply:
done + added null check